modules/cluster-telemetry.adoc

```diff
@@ -67,7 +67,7 @@
 ifndef::openshift-origin[]
 = Telemetry access for {product-title}
 
-In {product-title} {product-version}, the Telemetry service, which runs by default to provide metrics about cluster health and the success of updates, requires internet access. If your cluster is connected to the internet, Telemetry runs automatically, and your cluster is registered to the link:https://cloud.redhat.com/openshift[{cloud-redhat-com} (OCM)].
+In {product-title} {product-version}, the Telemetry service, which runs by default to provide metrics about cluster health and the success of updates, requires internet access. If your cluster is connected to the internet, Telemetry runs automatically, and your cluster is registered to the link:https://console.redhat.com/openshift[{cloud-redhat-com} (OCM)].
```

Comment:
We should edit 'OCM' to 'OpenShift Cluster Manager' - or 'Red Hat OpenShift Cluster Manager' If it's the first mention.

Also a link update for the Subscription doc -- https://access.redhat.com/documentation/en-us/subscription_central/2021/html/getting_started_with_the_subscriptions_service/con-how-to-select-datacollection-tool_assembly-requirements-and-your-responsibilities-ctxt#red_hat_openshift_monitoring_stack_and_other_tools_for_red_hat_openshift_data_collection